.github/workflows/pages.yml

```diff
@@ -4,7 +4,27 @@ on:
   # Runs on pushes targeting the default branch
   push:
     branches: ["main"]
-
+    paths:
+      - .github/workflows/pages.yml
+      - changelog/*
+      - docs/*
+      - example-notebooks/*
+      - src/seismometer/*
+      - Makefile
+      - pyproject.toml
+      - setup.cfg
+  
+  pull_request:
+    paths:
+      - .github/workflows/pages.yml
+      - changelog/*
```

Comment:
Nit: I think we can drop the changelog reference; it's currently not in scope of the doc build (to my knowledge) so won't represent a pages change.

The release PR will maybe have some edits to make on the constructed release_notes, but I think that's ok. We can revisit if it becomes burdensome.

Reply:
finc